bot/cogs/monstersurvey.py

```diff
@@ -0,0 +1,161 @@
+from discord.ext.commands import Bot, Context
```

Comment:
Please sort out the imports according to PEP8: https://www.python.org/dev/peps/pep-0008/#imports

3rd party libraries should be after built-ins, with a blank line in between. Most of the modern IDEs should have a command to do this for you.

```python
import json
import logging
import os
from asyncio import sleep as asleep
from typing import Optional

from discord import Embed
from discord.ext import commands
from discord.ext.commands import Bot, Context
```